homeassistant/components/mqtt/scene.py

```diff
@@ -103,23 +103,31 @@ class MqttScene(
 
     _entity_id_format = scene.DOMAIN + ".{}"
 
-    def __init__(self, hass, config, config_entry, discovery_data):
+    _config: ConfigType
```

Comment:
Is that necessary? Is it not already set on `MqttEntity`?
https://github.com/home-assistant/core/blob/a34148104acd3210932644fd33350f32f61d0789/homeassistant/components/mqtt/mixins.py#L1038